tests/driver_sdp3x/Makefile

```
@@ -0,0 +1,16 @@
+include ../Makefile.tests_common
+
+DRIVER ?= sdp3x
+
+USEMODULE += $(DRIVER)
+USEMODULE += xtimer
+USEMODULE += periph_i2c
+USEMODULE += printf_float
+USEMODULE += periph_gpio_irq
```

```suggestion
USEMODULE += printf_float
```

those should be handled by `Makefile.dep`